File: practices/financial_calculator.py
#SP 6th Financial calculator 

name = input("Hello please write your name").strip().title()
print("Hello", name,"welcome to Financial calculator!")

monthly_income = int(input("Please write your monthly income"))

monthly_rent = int(input("Please write your monthly rent"))

monthly_utilities = int(input("please write your monthly utilities"))

monthly_groceries = int(input("Please write your monthly groceries"))

monthly_transport = int(input("Please write your monthly transport"))

monthly_rentone = monthly_rent 
monthly_renttwo = monthly_rentone
monthly_renttwo /= monthly_income
monthly_renttwo *= 100

print("your monthly rent is",monthly_rent,"This is", round(monthly_renttwo),"% of your income")

monthly_utilitiesone = monthly_utilities
monthly_utilitiestwo = monthly_utilitiesone
monthly_utilitiestwo /= monthly_income
monthly_utilitiestwo *= 100

print("Your monthly utilities are",monthly_utilities,"This is",round(monthly_utilitiestwo),"% of your income")

monthly_groceriesone = monthly_groceries
monthly_groceriestwo = monthly_groceriesone
monthly_groceriestwo /= monthly_income
monthly_groceriestwo *= 100

print("Your monthly groceries are", monthly_groceries,"This is",round(monthly_groceriestwo),"% of your income")

monthly_transportone = monthly_transport
monthly_transporttwo = monthly_transportone
monthly_transporttwo /= monthly_income
monthly_transporttwo *= 100

print("Your monthly transport is",monthly_transport,"This is",monthly_transporttwo,"% of your income")

monthly_incomeone = monthly_income - monthly_rent - monthly_utilities - monthly_groceries - monthly_transport
print("You will have $",monthly_incomeone,"To spend every month!")
print("Thank you for using financial calculator",name)

File: c_programming/practices/financial_calculator.c
//SP 6th Financial Calculator 
#include <stdio.h>

int main(void){
    float monthly_income = 0;
    float monthly_rent = 0;
    float monthly_utilities = 0;
    float monthly_groceries = 0;
    float monthly_transport = 0;

    printf("Hello user welcome to financial calculator\n");
    printf("Please write your monthly income\n");
    scanf("%f", &monthly_income);
    
    printf("Please write your monthly rent\n");
    scanf("%f", &monthly_rent);

    printf("Please write your monthly utilies cost\n");
    scanf("%f", &monthly_utilities);

    printf("Please write your monthly groceries cost\n");
    scanf("%f", &monthly_groceries);

    printf("Please write your monthly transport cost\n");
    scanf("%f", &monthly_transport);

    float monthly_rent2 = monthly_rent/monthly_income*100;
    float monthly_utilities2 = monthly_utilities/monthly_income*100;
    float monthly_groceries2 = monthly_groceries/monthly_income*100;
    float monthly_transport2 = monthly_transport/monthly_income*100;

    printf("Your monthly rent is %2.2f and is %2.2f %% of your income\n", monthly_rent, monthly_rent2);
    printf("Your monthly rent is %2.2f and is %2.2f %% of your income\n", monthly_utilities, monthly_utilities2);
    printf("Your monthly rent is %2.2f and is %2.2f %% of your income\n", monthly_groceries, monthly_groceries2);
    printf("Your monthly rent is %2.2f and is %2.2f %% of your income\n", monthly_transport, monthly_transport2);


    float monthly_income2 = monthly_income*0.1;
    int amount_left = monthly_income-monthly_rent-monthly_utilities-monthly_groceries-monthly_transport;
    printf("You should save at least %2.2f and that is 10%% of your income\n", monthly_income2);
    printf("You have %d left to spend!",amount_left);



    return 0;
}